specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizerReceipt.json

```diff
@@ -0,0 +1,266 @@
+            "schema": {
+              "$ref": "Ocr.json#/definitions/ComputerVisionError"
+            }
+          },
+          "x-ms-examples": {
+            "Successful 'Get Read Receipt Result' request": {
+              "$ref": "./examples/SuccessfulReadReceiptResult.json"
+            }
+          }
+        }
+      }
+    }
+  },
+
+  "x-ms-paths": {
+    "/prebuilt/receipt/asyncBatchAnalyze?overload=stream": {
+      "post": {
+        "description": "Read Receipt operation. When you use the 'Batch Read Receipt' interface, the response contains a field called 'Operation-Location'. The 'Operation-Location' field contains the URL that you must use for your 'Get Read Receipt Result' operation.",
+        "operationId": "BatchReadReceiptInStream",
+        "parameters": [
+          {
+            "$ref": "Ocr.json#/parameters/ImageStream"
+          }
+        ],
+        "consumes": [
+          "application/octet-stream"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "202": {
+            "description": "The service has accepted the request and will start processing later. It will return 'Accepted' immediately and include an 'Operation-Location' header. Client side should further query the operation status using the URL specified in this header. The 'Operation-Location' URL will expire in 48 hours.",
+            "headers": {
+              "Operation-Location": {
+                "description": "URL to query for status of the operation. The URL will expire in 48 hours. ",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "Ocr.json#/definitions/ComputerVisionError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful 'Batch Read Receipt' request": {
+            "$ref": "./examples/SuccessfulBatchReadReceiptWithStream.json"
+          }
+        }
+      }
+    }
+  },
+
+  "definitions": {
+    "readReceiptResult": {
+      "type": "object",
+      "description": "Analysis result of the 'Batch Read Receipt' operation.",
+      "properties": {
+        "status": {
+          "description": "Status of the read operation.",
+          "$ref": "Ocr.json#/definitions/OperationStatus"
+        },
+        "recognitionResults": {
+          "description": "Text recognition result of the 'Batch Read Receipt' operation.",
+          "type": "array",
+          "items": {
+            "$ref": "Ocr.json#/definitions/TextRecognitionResult"
+          }
+        },
+        "understandingResults": {
+          "description": "Semantic understanding result of the 'Batch Read Receipt' operation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/understandingResult"
+          }
+        }
+      }
+    },
+
+    "understandingResult": {
```

Comment:
Not sure I understand the meaning of this model name. Please make the description a little clearer and/or rename this to a more intuitive name

Reply:
Thanks for noticing the missing documentation.  Sandeep, let's add:
A set of extracted fields corresponding to a semantic object, such as a receipt, in the input document.

Reply:
done.